pkg/expression/builtin_miscellaneous.go

```diff
@@ -57,6 +57,9 @@ var (
 	_ functionClass = &nameConstFunctionClass{}
 	_ functionClass = &releaseAllLocksFunctionClass{}
 	_ functionClass = &uuidFunctionClass{}
+	_ functionClass = &uuidv4FunctionClass{}
```

Comment:
Do you think we should also implement vectorized versions of these functions, in `builtin_miscellaneous_vec.go`?